terasoluna-tourreservation-domain/src/main/resources/META-INF/spring/terasoluna-tourreservation-codelist.xml

```diff
@@ -65,6 +65,7 @@
         <entry key="ja_JP" value-ref="tourDaysMapJa" />
       </util:map>
     </property>
+    <property name="fallbackTo" value="en" />
```

Comment:
- [ ] Good to set a fallback locale using Abstract parent bean.
Because the fallback locale should be unique in the system.

How do you think?

Reply:
I agree with your opinion.
Modified by e0a269dff3606bcff15f4a281c87407f0ee2622f